src/Cake.Common/IO/FileAliases.cs

```diff
@@ -331,5 +331,35 @@ public static FilePath MakeAbsolute(this ICakeContext context, FilePath path)
 
             return path.MakeAbsolute(context.Environment);
         }
+
+        /// <summary>
+        /// Gets the size of a file in bytes.
+        /// </summary>
+        /// <param name="context">The context.</param>
+        /// <param name="filePath">The path.</param>
+        /// <returns>Size of file in bytes or -1 if file doesn't exist.</returns>
+        /// <example>
+        /// <code>
+        /// Information("File size: {0}", FileSize("./build.cake"));
+        /// </code>
+        /// </example>
+        [CakeMethodAlias]
+        [CakeAliasCategory("Exists")]
+        public static long FileSize(this ICakeContext context, FilePath filePath)
+        {
+            if (context == null)
+            {
+                throw new ArgumentNullException("context");
+            }
+
+            if (filePath == null)
+            {
+                throw new ArgumentNullException("filePath");
+            }
+
+            var file = context.FileSystem.GetFile(filePath.MakeAbsolute(context.Environment));
+
+            return file.Exists ? file.Length : -1;
```

Comment:
Personally I think that this should throw a FileNotFoundException if the file does not exist (just like [System.IO.File.Length](https://msdn.microsoft.com/en-us/library/system.io.fileinfo.length(v=vs.110).aspx)).